### PR TITLE
Anti-personnel landmines no longer trigger on dead mobs

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Any mob thats dead can no longer trigger landmines

## Why It's Good For The Game
The amount of times i've seen people blow themselves trying to sell dead xenos and walking through a mind-field is unreal.
I don't see why we should keep this either , it only serves to make landmines even worse(like they aren't already quite niche and weak)

## Changelog
:cl:
balance: Anti-personnel landmines no longer get triggered by dead mobs
/:cl:

### CODE REVIEW

**Comment by QualityVan** [comment]
Better to just not trigger on dead things in general, no?

**Comment by Naaanii** [comment]
No longer will people dragging corpses destroy the FOB.

File: code/game/objects/items/explosives/mine.dm [hunk 1]
```diff
@@ -202,6 +202,10 @@ Stepping directly on the mine will also blow it up
 		return
 
 	if(linked_mine && isliving(A))
+		var/mob/living/unlucky_person = A
+		// Don't trigger for dead xenos
+		if(isxeno(unlucky_person) && unlucky_person.stat == DEAD)
```
Comment (BraveMole) [inline code/game/objects/items/explosives/mine.dm:207]: why the isxeno check ?

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/objects/items/explosives/mine.dm (hunk 1 above)
Comment (MLGTASTICa) [reply code/game/objects/items/explosives/mine.dm:207]: i don't wannna affect HvH balance where you can throw people at landmines to detonate them ? or should it also not trigger for anything dead in general?

Comment (MLGTASTICa) [reply code/game/objects/items/explosives/mine.dm]: made it not trigger for anything dead.

**Review by BraveMole — APPROVED** [review]

**Comment by QualityVan** [comment]
If I was being finicky I'd move the check up to inside trip_mine since that's where the rest of the triggering mob's conditions are checked, but that's not super important.

**Comment by Lumipharon** [comment]
Boo, I will  no longer be able to throw dead marines into minefields to blow them up in combat patrol.

Probs good change though.

**Comment by Al-1ce** [comment]
hint for all you aspiring robustos out there: you can disable these with a multitool instead of being a murderous psychopath

**Pull request merged**